src/main/java/com/fasterxml/jackson/databind/ext/DOMDeserializer.java

```diff
@@ -35,18 +35,18 @@
             parserFactory.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);
         } catch(ParserConfigurationException pce) {
             // not much point to do anything; could log but...
-        } catch (Error e) {
+        } catch (Exception e) {
             // 14-Jul-2016, tatu: Not sure how or why, but during code coverage runs
             //   (via Cobertura) we get `java.lang.AbstractMethodError` so... ignore that too
         }
 
         // [databind#2589] add two more settings just in case
         try {
             parserFactory.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
-        } catch (Throwable t) { } // as per previous one, nothing much to do
+        } catch (Exception t) { } // as per previous one, nothing much to do
```

Comment:
In this case need to leave `Exception` like here, for unsupported feature.

(or could consider catching `ParserConfigurationException` -- but for now I prefer `Exception`).

Reply:
Exactly. I do not (alas!) trust them NOT to throw, say, `IllegalArgumentException` (or NPE as actually documented).
And I say this as maintainer of Woodstox which tries to follow API spec but which still occasional has differed (esp. where spec was incomplete).